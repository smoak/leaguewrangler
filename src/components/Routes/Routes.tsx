import React, { SFC } from 'react';
import { BrowserRouter, Redirect, Route, Switch } from 'react-router-dom';

import PageHome from '../PageHome';

const Routes: SFC = () => {
  return (
    <BrowserRouter>
      <Switch>
        <Route exact={true} path="/home" component={PageHome} />
        <Redirect to="/home" />
      </Switch>
    </BrowserRouter>
  );
};

export default Routes;
