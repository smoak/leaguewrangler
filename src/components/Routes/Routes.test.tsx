import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import Routes from '.';

describe('Routes', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<Routes />);
  });

  it('renders', () => {
    expect(component).toMatchSnapshot();
  });
});
