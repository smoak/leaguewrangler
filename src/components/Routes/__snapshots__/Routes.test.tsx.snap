// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes renders 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/home"
    />
    <Route
      component={[Function]}
      path="/login"
    />
    <Redirect
      push={false}
      to="/home"
    />
  </Switch>
</BrowserRouter>
`;
